aries-site/src/pages/components/accordion.js

```diff
@@ -18,23 +19,72 @@ const Accordion = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
-        <SubsectionText>
-          The accordion affords content to be delivered progressively. The
-          chevron is used to identify the expand or collapse action while the
-          entire header can be clicked to expand or collapse content.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
         <Example
-          details={[
-            `When seeking to provide maximum content in limited, vertical 
-            space, an accordion is a good alternative.`,
-          ]}
           docs="https://v2.grommet.io/accordion#props"
           code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/AccordionExample.js"
+          figma="https://www.figma.com/file/UvTBUQdhk07wV79nijsrtr/HPE-Accordion-Component?node-id=71%3A25"
         >
           <AccordionExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          When seeking to provide maximum content in limited, vertical space, an
+          accordion is a good alternative.
+        </SubsectionText>
+        <SubsectionText>
+          An accordion is a group of vertically stacked collapsible buttons with
```

Comment:
Maybe:
An accordion is a group of vertically stacked collapsible buttons with title headers. The titles give the user a high-level idea of what content will be disclosed when an accordion panel opens.